### ISSUE TITLE
What about Sentinel?

### ISSUE THREAD

**Issue by DeanGross** [MicrosoftDocs/memdocs#3414]
The documentation mentions integration with 3rd party SIEMs, but does not even mention Microsoft Sentinel, which should be the first option. Please add this to the instructions. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b1cee1ad-3f38-5bde-09a3-32742d8d39cd
* Version Independent ID: bf4cf7d8-e162-80ec-98b0-e46b452e44cb
* Content: [Route logs to Azure Monitor using Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/fundamentals/review-logs-using-azure-monitor)
* Content Source: [memdocs/intune/fundamentals/review-logs-using-azure-monitor.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/fundamentals/review-logs-using-azure-monitor.md)
* Service: **microsoft-intune**
* Sub-service: **fundamentals**
* Technology: ****
* GitHub Login: @MandiOhlinger
* Microsoft Alias: **mandia**

**Comment by yogkumgit** [comment]
@MandiOhlinger Please help us in resolving this issue. Thanks

**Comment by MandiOhlinger** [comment]
@DeanGross Hey - We have [Integrate SentinelOne with Intune](https://learn.microsoft.com/mem/intune/protect/sentinelone-mtd-connector-integration). We can link to that article. Is that what you're thinking? It'd be great if you could provide more details.

**Comment by yogkumgit** [comment]
@DeanGross

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!

**Comment by DeanGross** [comment]
No, I was not asking about SentinelOne, I was referring to Microsoft Sentinel that is in Azure.
